src/target/riscv/riscv-013.c

```diff
@@ -4872,6 +4873,16 @@ static int write_memory(struct target *target, target_addr_t address,
 	return ret;
 }
 
+static bool riscv013_get_impebreak(const struct target *target)
+{
+	return get_info(target)->impebreak;
+}
+
+static unsigned int riscv013_get_progbufsize(const struct target *target)
+{
+	return get_info(target)->progbufsize;
+}
```

Comment:
If you like:

RISCV013_INFO() is more frequenly used than get_info(). So for code consistency, you may consider this:

```suggestion
static bool riscv013_get_impebreak(const struct target *target)
{
	RISCV013_INFO(info);
	return info->impebreak;
}

static unsigned int riscv013_get_progbufsize(const struct target *target)
{
	RISCV013_INFO(info);
	return info->progbufsize;
}
```

Reply:
Addressed

Reply:
It looks like this change has not been pushed - can you please double-check it? Thanks.

Reply:
You are right. Updated commit